### PR TITLE
patch: document vm apis

### PR DESCRIPTION
**Pull request by pelikhan**
- [ ] @mmoskal look at the TODOs, there's a few APIs I am unsure about how they are used or if useful

### CODE REVIEW

File: runtime/devicescript-vm/dist/types.d.ts [hunk 1]
```diff
@@ -11,25 +11,103 @@ export declare type DevsModule = EmscriptenModule & typeof Exts & {
...
     /**
      * Verifies the format and version of the bytecode
      * @param binary DeviceScript bytecode
-     * @returns error code, 0 if verificatino is successful
+     * @returns error code, 0 if verification is successful
      */
     function devsVerify(binary: Uint8Array): number;
+    /**
+     * Deploys a DeviceScript bytecode to the virtual machine
+     * @param binary
```
Comment (mmoskal) [inline runtime/devicescript-vm/dist/types.d.ts:81]: This deploys to some other attached VM but to to self

Comment (pelikhan) [reply runtime/devicescript-vm/dist/types.d.ts:81]: How do you specify which other vm?

Comment (mmoskal) [reply runtime/devicescript-vm/dist/types.d.ts:81]: you don't; deploys to the first one it finds.

Comment (mmoskal) [reply runtime/devicescript-vm/dist/types.d.ts:81]: I mean it's a hack...

Comment (pelikhan) [reply runtime/devicescript-vm/dist/types.d.ts:81]: Marked as @internal

**Pull request merged**